recipes/cpython/all/conanfile.py

```diff
@@ -692,47 +684,45 @@ def package_info(self):
                 self.cpp_info.components["_hidden"].requires.append("sqlite3::sqlite3")
             if self.options.get_safe("with_curses", False):
                 self.cpp_info.components["_hidden"].requires.append("ncurses::ncurses")
-            if self.options.get_safe("with_bsddb"):
-                self.cpp_info.components["_hidden"].requires.append("libdb::libdb")
             if self.options.get_safe("with_lzma"):
                 self.cpp_info.components["_hidden"].requires.append("xz_utils::xz_utils")
             if self.options.get_safe("with_tkinter"):
                 self.cpp_info.components["_hidden"].requires.append("tk::tk")
+            self.cpp_info.components["_hidden"].includedirs = []
             self.cpp_info.components["_hidden"].libdirs = []
+            if self.settings.os in ["Linux", "FreeBSD"]:
+                self.cpp_info.components["_hidden"].system_libs.append("nsl")
 
         if self.options.env_vars:
             bindir = os.path.join(self.package_folder, "bin")
-            self.output.info("Appending PATH environment variable: {}".format(bindir))
+            self.output.info(f"Appending PATH environment variable: {bindir}")
             self.env_info.PATH.append(bindir)
```

Comment:
```suggestion
            self.env_info.PATH.append(bindir)
            self.buildenv_info.prepend_path("PATH", bindir)
            self.runenv_info.prepend_path("PATH", bindir)
```

Env info is Conan 1.x and will be deprecated in the future. In Conan 2.x you should use runenv or buildenv.For Python I would say both cases are valid because I see python scripts during build time, only to generate C/C++ code, but also I saw some packaged tools that are python scripts too.

Reply:
I added these for all the existing variables, as well as a note regarding the old version's removal at some point.